### PR TITLE
Read in assingeelist.json

### PR DESCRIPTION
**Pull request by Kyl-Bempah**
Initial implementation of assignment list for automatic issue allocation.

### CODE REVIEW

File: generators/GitHubIssueGenerator.py [hunk 1]
```diff
@@ -230,7 +241,15 @@ def open_github_issue(self):
                 except UnknownObjectException as ex:
                     print(f"Couldn't find GitHub Tag {tag}, skipping and continuing.", file=sys.stderr, flush=False)
                     pass
-            _issue = repo.create_issue(self.generate_issue_title(), body=_message, labels=_github_tags_objects)
+            _assignees=[]
+            for tag in _tags:
+                try:
+                    if tag in self.assigneelist:
+                        _assignees.append(org.get_members(self.assigneelist[tag]))
```
Comment (Kyl-Bempah) [inline generators/GitHubIssueGenerator.py:248]: Does this look right in terms of obtaining a github user object?

Comment (schmidtd) [reply generators/GitHubIssueGenerator.py:248]: `org.get_members` will return a list of all users in the org (no matter what the argument is).

Comment (Kyl-Bempah) [reply generators/GitHubIssueGenerator.py]: How about my latest push?

**Review by gurnben — APPROVED** [review]
LGTM - thanks for the good implementation work - this fits the mold with previous work so it'll be easier to understand/maintain!

**Review by gurnben — APPROVED** [review]
LGTM!

**Pull request merged**